sdk/translation/Azure.AI.Translation.Document/tests/TranslationOperationLiveTests.cs

```diff
@@ -390,6 +390,34 @@ public async Task ExistingFileInTargetContainer()
             Assert.AreEqual(new DocumentTranslationErrorCode("TargetFileAlreadyExists"), documentsList[0].Error.ErrorCode);
         }
 
+        [TestCase("Foo Bar", typeof(ArgumentException))]
+        [TestCase("", typeof(ArgumentException))]
+        [TestCase(null, typeof(ArgumentNullException))]
+        public void DocumentTranslationOperationWithInvalidGuidTest(string invalidGuid, Type expectedException)
+        {
+            var client = GetClient();
+            Assert.Throws(expectedException, () => new DocumentTranslationOperation(invalidGuid, client));
+        }
+
+        [TestCase("Foo Bar", typeof(ArgumentException))]
+        [TestCase("", typeof(ArgumentException))]
+        [TestCase(null, typeof(ArgumentNullException))]
+        public async Task GetDocumentStatusWithInvalidGuidTest(string invalidGuid, Type expectedException)
```

Comment:
Test look great. now u need to record all of them and add them to the PR so the CI can pass